src/instance/minecraft/chat/RepeatChat.ts

```diff
@@ -6,20 +6,19 @@ import { CommandsManager } from '../CommandsManager'
 import { EventType } from '../../../common/ApplicationEvent'
 
 const MESSAGES = [
-  "can't repeat same message...",
-  'I wish I had a way to repeat same messages over and over again :(',
-  'Hypixel blocked this message for repeating... again! D:',
-  "Hold down your horses, can't say same message twice!",
-  "twinkle twinkle little star, can't repeat message with big R",
+  "Can't repeat the same message...",
+  'I wish I had a way to repeat the same messages over and over again :(',
+  'Hypixel blocked this message for repeating... Again! D:',
+  "Hold your horses, can't say same message twice!",
+  "Twinkle twinkle little star, can't repeat message with big R",
   'No, no, no, NO. no message repetition D:',
   "RIP, can't say same thing twice",
-  "Wonder where the message have gone? Yeah... can't repeat it :P",
-  'Message can not be repeated!',
+  "Wonder where the message has gone? Yeah... Can't repeat it :P",
+  'Message cannot be repeated!',
   'The verdict has been given and will not be repeated!',
-  "I once tried to say something twice, but then hypixel gave me the L and denied my privilege. Now I can't say the same message twice. Rest in peace my thought.",
```

Comment:
Rest in peace that message. Any reason why completely removing it?

Reply:
Far too long, would nearly fill the entire chat over 1 error